Simplenote/AboutViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  AboutViewController.swift
+//  Simplenote
+//
+
+import Cocoa
+
+class AboutViewController: NSViewController {
+    let blogUrl =       "https://simplenote.com/blog"
+    let twitterUrl =    "https://twitter.com/simplenoteapp"
+    let appsUrl =       "https://simplenote.com"
+    let githubUrl =     "https://github.com/automattic/simplenote-macos"
+    let hiringUrl =     "https://automattic.com/work-with-us"
+    let privacyUrl =    "https://simplenote.com/privacy/"
+    let termsUrl =      "https://simplenote.com/terms/"
```

Comment:
Another Swift trick!

```
struct Constants {
   static let blogUrl = "..."
}
```

(No need to change anything, this is perfect the way it is, just passing along few minor things we do around!)

Reply:
That's... beautiful 😍. Updated to that struct pattern. Mmmm reminds me of tasty Java :)